### DISCUSSION TITLE
Can the multiple diffraction paths and the diameter of reflection plus diffraction be simulated?

### DISCUSSION THREAD

**Discussion by vcyycv0** [NVlabs/sionna#487] [Q&A] [answered]
Dear all,
I'm doing an indoor ray tracing simulation by Sionna. I wonder if Sionna can simulate multiple diffraction paths (Tx-Diff-Diff-Rx) and paths that have both reflection and diffraction(Tx-refl-diff-Rx or Tx-diff-refl-Rx).

**Comment by jhoydis** [comment, marked as answer]
Hi @vcyycv0,

In the current version, only first-order diffraction is implemented. You can only have TX-Diff-RX.

**Comment by vcyycv0** [comment]
thank you for your answer. This is a good explanation for some of the path discontinuities in the antenna whole row simulation results. And will the Tx-Diff-Diff-Rx , Tx-refl-diff-Rx and Tx-diff-refl-Rx paths be added in future releases?
